### ISSUE TITLE
Postman 7.11.0, 7.1.1 and 7.10.0 can't start on Windows 10

### ISSUE THREAD

**Issue by Syslog2905** [a85/POSTMan-Chrome-Extension#7693]
**Describe the bug**
Postman 7.11.0, 7.1.1 and 7.10.0 can't start on Windows 10

**To Reproduce**
1. Install Postman
2. Start it from Windows start menu or Run as Administrator ( does not matter)
3. Error pops up: "Please contact support"
4. Postman hangs on Preparing your workspace
**Expected behavior**
To start normally
**Screenshots**

**App information (please complete the following information):**
 - App Type [e.g.  Native App]
 - Postman Version [e.g. 7.1.1,7.10.0, 7.11.0)
 - OS: Win 10, 64bits

**Additional context**
It happens after update. The posible reason might be that th eprogram even can't make clean uninstall - it leaves files under C:\Users\{user}\AppData\Roaming\Postman
![postman](https://user-images.githubusercontent.com/12511295/69426544-2b0eb280-0d36-11ea-809e-4d9ff208d471.PNG)

 Please test better Postman next time!

**Comment by Syslog2905** [comment]
[pluginHost.log](https://github.com/postmanlabs/postman-app-support/files/3879540/pluginHost.log)
[renderer-auth.log](https://github.com/postmanlabs/postman-app-support/files/3879541/renderer-auth.log)
[renderer-console.log](https://github.com/postmanlabs/postman-app-support/files/3879542/renderer-console.log)
[renderer-requester.log](https://github.com/postmanlabs/postman-app-support/files/3879543/renderer-requester.log)
[renderer-runner.log](https://github.com/postmanlabs/postman-app-support/files/3879544/renderer-runner.log)
[renderer-shared.log](https://github.com/postmanlabs/postman-app-support/files/3879545/renderer-shared.log)
[renderer-shell.log](https://github.com/postmanlabs/postman-app-support/files/3879546/renderer-shell.log)
[main.log](https://github.com/postmanlabs/postman-app-support/files/3879547/main.log)

**Comment by ankurdengla** [comment]
The Postman app is built on Electron. We had updated the electron version from 1.8.8 to 3.1.8 from Postman 7.3.4 onwards. The error indicated by the logs can happen in the case when -
* User launches an app with electron 3.1.8 (in this case app 7.9.0)
* Then user launches app version with electron 1.8.8 (7.1.1 in this case)
* User again launches app with electron 3.1.8 

This causes inconsistencies in the underlying storage of the app and prevents the app from launching. User can unblock themselves by following this troubleshooting guide - https://support.getpostman.com/hc/en-us/articles/360025359014-How-to-resolve-the-Could-not-open-Postman-error-

We are tracking this here - https://github.com/postmanlabs/postman-app-support/issues/6723

Closing this as a duplicate.
